src/callbacks_wandb.py

```diff
@@ -235,3 +235,4 @@ def on_validation_end(
                 axs[1, i].axis("off")
 
             self.logger.experiment.log({"Images": wandb.Image(fig)})
+            plt.close("all")
```

Comment:
Could we do `plt.close(fig)` here to be more explicit what we are closing?

Reply:
I was trying to be extra cautious and class everything, but I´ve tested and there´s no difference. Changed.